appengine/helloworld-new-plugins/README.md

```diff
@@ -0,0 +1,59 @@
+# Google App Engine Standard Environment Hello World Sample
+
+This sample demonstrates how to deploy an application on Google App Engine.
+
+See the [Google App Engine standard environment documentation][ae-docs] for more
+detailed instructions.
+
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Setup
+
+Use either:
+
+* `gcloud init`
+* `gcloud beta auth application-default login`
+
+## Maven
+### Running locally
+
+    $ mvn appengine:run
+  
+### Deploying
+
+    $ mvn appengine:deploy
```

Comment:
Not a problem with your code, but it's annoying that they used the same name for the plugin. That has a lot of potential for confusion as to which plugin someone is using.